src/mock/mockRoutes.ts

```diff
@@ -0,0 +1,20 @@
+import faker from "faker";
+
+//Custom Generator
+const routeGenerator = () => ({
+  name: faker.unique(faker.internet.url),
+  route: faker.unique(faker.system.filePath),
+  status: faker.datatype.boolean(),
+});
+
+//Generator Function
```

Comment:
generates routes of a given number

in js, a generator function begins w a * prefix

Reply:
Hey actually it's not completely the same as the Js generator function but kind of a nomenclature used by me for this type of function

Reply:
What are your thoughts can we call it something else other than a custom generator? to avoid confusion

Reply:
custom generator is still not how you should name it